### DISCUSSION TITLE
PKsim - how to add binding factor for each organ

### DISCUSSION THREAD

**Discussion by jyna6** [Open-Systems-Pharmacology/Forum#1843] [Questions & Problems]
Hello,

If a drug shows high binding to a brain (not to other organs), how can I give this information in PKsim?
This drug has different fu values in plasma and brain, and I have no idea how to input this into my PKsim PBPK model.

Thank you in advance :-)

**Comment by Proen1** [comment]
Hi jyna6, what do you mean high binding?  Higher partition coefficient?  Based on lipophlicity, pKa nd the type of algorithm you choose , PK-Sim will calculate the tissue partition coefficients.  If you think the partition to brain is underprediccted you can change directly in the simulation in parameters->distribution-> intracellular-plasma. 
![image](https://github.com/user-attachments/assets/3a083c69-c66d-47d7-b834-8b002cf51fa1)
